packages/drivers/odsp-driver/src/createNewUtils.ts

```diff
@@ -253,8 +253,10 @@ export async function createNewFluidContainerCore<T>(args: {
 					`\r\n--${formBoundary}--`;
 
 				let postBody = snapshotBody;
+				// We use the byte length of the post body to determine if we should use the multipart/form-data or not. This helps
+				// in cases where the body contains data with different language where 1 char could be multiple code points.
 				if (
-					postBodyWithAuth.length <= maxUmpPostBodySize &&
+					new TextEncoder().encode(postBodyWithAuth).length <= maxUmpPostBodySize &&
```

Comment:
Is `new TextEncoder().encode(...)` what is used to actually prepare the body later?  Can we move this check or otherwise unify those two codepaths, to avoid these two parts becoming out of sync again?

Reply:
Not that I expect it to "get out of sync", but the fact that this code doesn't have in hand the actual bytes for the POST body is what made this bug possible/easy to begin with.